nep-29.mediawiki

```diff
@@ -0,0 +1,41 @@
+NEP: 29
+Title: Standardization of NEP Number Assignment and Usage
+Author: Jimmy Liao <jimmy@r3e.network>
+Type: Meta
+Status: Draft
+Created: 2024-05-22
+
+==Abstract==
+
+NEP 29 proposes a standardized method for assigning and managing Neo Enhancement Proposal (NEP) numbers. This proposal addresses the logistical challenges involved with the naming and referencing of NEP drafts, which currently cannot be assigned a permanent NEP number until the proposal is officially adopted. The solution is to uniquely assign NEP numbers at the draft stage, which will remain permanently associated with the draft, whether or not it is adopted.
```

Comment:
But what happens if some NEP is declined? Right now we have consecutive numbers of the accepted NEPs, and it would be nice to preserve it.

Reply:
we have relatively `consecutive` numbers, some of them are already deprecated, no issue at all. We just list those active NEPs. Declined draft can drop that NEP number. No one will be interested why there is an NEP missing number, people will only focus on NEPs that are of their use.